pydicom/filereader.py

```diff
@@ -912,28 +912,57 @@ def data_element_offset_to_value(is_implicit_VR, VR):
     return offset
 
 
-def read_deferred_data_element(fileobj_type, filename, timestamp,
+def read_deferred_data_element(fileobj_type, filename_or_obj, timestamp,
                                raw_data_elem):
     """Read the previously deferred value from the file into memory
-    and return a raw data element"""
+    and return a raw data element.
+
```

Comment:
Suggest adding a comment that this is normally not called in user code, but automatically by pydicom if needed.

Reply:
Good point - maybe we should remove it from the public API (e.g. from `fileio.read.rst`) in this case?

Reply:
Yes, possibly - I think we should at some point (for v2.0?) take a good look at what is in the public API and really pair down on the number of functions exposed there.  For now, let's just merge this as is.

Reply:
Yes, good idea!